libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -4507,6 +4508,30 @@ void GCS_MAVLINK::send_attitude() const
         omega.z);
 }
 
+void GCS_MAVLINK::send_attitude_quaternion() const
+{
+    const AP_AHRS &ahrs = AP::ahrs();
+    Quaternion quat;
+    if (!ahrs.get_secondary_quaternion(quat)) {
+        return;
+    }
+    const float repr_offseq_q[] {0,0,0,0};  // unused, but probably should correspond to the AHRS view?
+    const Vector3f omega = ahrs.get_gyro();
+    gcs().send_text(MAV_SEVERITY_WARNING, "quatin: %f %f %f %f", quat.q1, quat.q2, quat.q3, quat.q4);
```

Comment:
I guess this is a bit of leftover debug?

Reply:
Huh.  Whoops.

Yes, that was just making sure I was getting out the other end what I was putting in.